proto/regen/data/v1/state.proto

```diff
@@ -62,6 +62,7 @@ message Resolver {
     primary_key : {fields : "id", auto_increment : true}
     index : {id : 1, fields : "url"}
     index : {id : 2, fields : "manager"}
```

Comment:
was wondering if the new index below allows us to remove any of these other ones, and i couldn't seem to find any with the `manager` index being used. is it still needed?

Reply:
Looks like it wasn't being used. Removed in https://github.com/regen-network/regen-ledger/pull/1219/commits/73aeb6ea45818c7b2f30d7b85ba6eac6bc873e4a.